.github/workflows/unit-tests-workflow.yml

```diff
@@ -47,4 +47,8 @@ jobs:
       - name: Run tests
         run: |
           cd OpenSearch-Dashboards/plugins/index-management-dashboards-plugin
-          yarn run test:jest
+          yarn run test:jest --coverage
+      - name: Uploads coverage
+        uses: codecov/codecov-action@v1
+        with:
+          token: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
Did you add a CODECOV token to the repo?

Reply:
We need to ask people from OpenSearch team to add it if there isn't.